pythran/pythonic/numpy/partial_sum.hpp

```diff
@@ -40,9 +68,8 @@ namespace pythonic
       const long count = expr.flat_size();
       types::ndarray<typename dtype::type, 1> the_partial_sum{
           types::make_tuple(count), __builtin__::None};
-      auto const p = utils::neutral<Op, typename E::dtype>::value;
-      auto iter = the_partial_sum.begin();
-      _partial_sum<Op, E::value>{}(expr, iter, p);
```

Comment:
It is still present for reduct but it will be my next step as reduction is also a ufunc method.